sbin/autograder/grade_item_main_runner.py

```diff
@@ -169,6 +169,11 @@ def executeTestcases(complete_config_obj, tmp_logs, tmp_work, queue_obj, submiss
                     # Move the files necessary for grading (runner, inputs, etc.) into the testcase folder.
                     setup_folder_for_grading(testcase_folder, tmp_work, job_id, tmp_logs,testcases[testcase_num-1])
                     my_testcase_runner = os.path.join(testcase_folder, 'my_runner.out')
+
+                    my_display="0"
+                    if "DISPLAY" in os.environ.keys():
+                        my_display = str(os.environ['DISPLAY'])
```

Comment:
Already merged, but you could condense the three lines into: `my_display = str(os.environ.get('DISPLAY', '0'))`